tests/test_units.py

```diff
@@ -0,0 +1,130 @@
+import pint
+import pytest
+
+from pymorize.units import to_caret_notation, convert
```

Comment:
If you change the `convert`'s function name you'll also have to change it in the tests.

Reply:
done.